src/main/java/nl/dtls/fairdatapoint/api/controller/exception/ExceptionControllerAdvice.java

```diff
@@ -72,6 +72,7 @@ public class ExceptionControllerAdvice {
     )
     public ErrorDTO handleBadRequest(Exception e) {
         log.warn(e.getMessage());
+        e.printStackTrace();
```

Comment:
Do we print these to stdout, or do we need a `log.debug` approach? (same for the others in this class)